sleipner/FIPNUM.INC

```diff
@@ -0,0 +1,26 @@
+--  Copyright 2019 The Sleipner Group.
+--  See the README file for the list of copyright owners making up the Sleipner Group.
```

Comment:
If you are adding copyright for this file it would be more correct to use Norce since this is  my own doing.